qml/harbour-fernschreiber.qml

```diff
@@ -30,6 +30,14 @@ ApplicationWindow
     cover: Qt.resolvedUrl("pages/CoverPage.qml")
     allowedOrientations: defaultAllowedOrientations
 
+    onApplicationActiveChanged: {
+        if(applicationActive) {
+            tdLibWrapper.handleForeground()
+        } else {
+            tdLibWrapper.handleBackground()
+        }
+    }
+
```

Comment:
There's no need to involve QML in this, the native `TDLibWrapper` object can directly receive `QGuiApplication::applicationStateChanged` signals from `qGuiApp`.